addons/storyshots/storyshots-core/package.json

```diff
@@ -108,6 +108,9 @@
     "@storybook/angular": {
       "optional": true
     },
+    "@storybook/react": {
```

Comment:
@Hypnosphi Should we also make `react-dom` optional at the same time?

Reply:
it is

Reply:
Note to self: don't review PRs before coffee. Thanks!